projects/wmf_enwiki.yaml

```diff
@@ -65,7 +65,7 @@ paths:
                   options:
                     response_cache_control: '{{options.purged_cache_control}}'
                     cx_host: '{{options.transform.cx_host}}'
-                - path: v1/mobileapps.yaml
+                - path: v1/mobileapps.js
```

Comment:
The /sys hierarchy should be removed too (in all old-style projects)

Reply:
done